docroot/themes/humsci/su_humsci_admin/scss/humsci_admin.portal.scss

```diff
@@ -0,0 +1,28 @@
+.content-header {
+  background: transparent;
+  text-align: center;
+}
+
+.breadcrumb {
+  display: none;
```

Comment:
Is there a better way to hide this? Can you not remove it from the layout?

Reply:
its the most simple way. all that it contains is a link to the home page. which the logo does anyways.

Reply:
Does this introduce any accessibility issues? Off the top of my head, I don't think so but it does introduce a DX issue. I prefer to remove the breadcrumb via config or template as that is the most obvious place the next developer will look.